### DISCUSSION TITLE
Share Your Froggy Frames

### DISCUSSION THREAD

**Discussion by tmm** [wevm/frog-archived#22] [Show and tell]
Share your Frames that use Frog. If they are OSS, drop a link so others can check them out!

**Comment by taofeek-ob** [comment]
A multi page frame of love story.

Frame: https://farcaster-frame-2.vercel.app/api

Code: https://github.com/taofeek-ob/farcaster-frame/

**Comment by realfakepicnic** [comment]
A crystal ball for the future of the /books channel 🔮

Frame: https://warpcast.com/picnic/0x688b3d00

Code: https://github.com/realfakepicnic/books-outcomes

**Comment by earth2travis** [comment]
⚔️ Adventure into [Raid Guild](https://www.raidguild.org/)

- Frame: [https://frame-guild-character-quest.vercel.app/api](https://frame-guild-character-quest.vercel.app/api)
- Code: [https://github.com/raid-guild/frame-guild-character-quest](https://github.com/raid-guild/frame-guild-character-quest)

**Comment by dalechyn** [comment]
XKCD frames
- Frame: https://xkcd-frame.xyz/api/post/203
- Code: https://github.com/dalechyn/xkcd-frame

**Comment by applefather-eth** [comment]
colletion of casts

https://warpcast.com/huangv/0xb11e24cc

**Reply by earth2travis** [reply]
Do you have a link to the repo?

**Comment by ggomaeng** [comment]
buy/sell frame for mint.club

https://warpcast.com/undefined/0x17c0e789

**Reply by earth2travis** [reply]
Do you have a link to the repo?

**Comment by onchainyaotoshi** [comment]
FRAME POLL FOR NFTOSHIS HOLDER

Frame: https://poll.cool/frame
Code: https://github.com/onchainyaotoshi/frame-poll

**Comment by earth2travis** [comment]
[YEET](https://yeet.haus/) from Frames

- Frame*: https://frames.yeet.haus/api/yeeter/0xda4a1616F7D2cfdA9b920A2ce111830001Df03Eb
- Code: https://github.com/yeet-haus/frames

`*` These are time sensitive based on having an active crowdfunding campaign

**Comment by onchainyaotoshi** [comment]
A warpcast action for Tip using $toshi with smart account wallet

Frame: https://tip-toshi.replit.app
Code: https://github.com/onchainyaotoshi/tip-toshi
Install: https://warpcast.com/~/add-cast-action?url=https://tip-toshi.replit.app/tip

**Comment by onchainyaotoshi** [comment]
A Warpcast action for tipping with $TOSHI v2, eliminating the need to top up the smart account wallet first!

Frame: https://tip-toshi-v2.replit.app/
Code: https://github.com/onchainyaotoshi/tip-toshi-v2
Install: https://warpcast.com/~/add-cast-action?url=https://tip-toshi.replit.app/tip2

**Comment by roberpkr** [comment]
The current frame takes the top 5 ERC20 tokens from Ethereum and Binance Smart Chain over the last 48 hours, sorted by popularity (number of transactions)

Planning to use image handle to add more context.

https://warpcast.com/roberpkr/0x58ded219

Thank you!

**Comment by lusorio** [comment]
Quidli channel/cast drops

https://warpcast.com/ahn.eth/0x56769275

Thanks to frog!

**Comment by thisyearnofear** [comment]
Frame for Gashawk.io

https://warpcast.com/papa/0x93e31852

Thanks for the froggy frog

**Comment by uratmangun** [comment]
https://warpcast.com/koisose/0x9e727fa4 farcastle attack reason decentralize ai frame
